src/transform2d.rs

```diff
@@ -18,51 +18,51 @@ use trig::Trig;
 use std::fmt;
 
 define_matrix! {
```

Comment:
I wonder if we even still need this, given now serde_derive is stable and comes with knobs to change the default deriving behaviour, we should look into simplifying it before 1.0.